logger.go

```diff
@@ -34,15 +33,10 @@ type Shuttle struct {
 
 // NewShuttle creates a shuttle client from a Config.
 func NewShuttle(cfg *LoggerConfig) (*Shuttle, error) {
-	token, ok := cfg.URL.User.Password()
-	if !ok {
-		return nil, errors.New("no logplex password provided")
-	}
-
 	// Create shuttle config.
 	conf := shuttle.NewConfig()
 	conf.LogsURL = cfg.URL.String()
-	conf.Appname = token
+	conf.Appname = "app"
```

Comment:
Changed this to `"app"` in light of https://github.com/heroku/pg_logplexcollector/pull/10#discussion_r57484606; however, still wondering if this should be configurable or will it always be `"app"` for DoD use cases?